client/src/pages/CreateWorkout/components/ListExercises/ListExercises.jsx

```diff
@@ -71,7 +87,13 @@ const ListExercises = () => {
           </div>
         </div>
       )}
-      {showAddForm && <ExerciseForm handleCloseForm={handleCloseForm} />}
+      {showAddForm ? (
```

Comment:
Haha these 2 are exact opposites of each other so you can combine them into one:

`{showAddForm ? <ExerciseForm ... /> : <Add exercise button />`